packages/quick-start-browser/src/CreateTable.tsx

```diff
@@ -0,0 +1,61 @@
+import { Button } from "@fluentui/react-components";
+import { Client } from "azure-kusto-data";
+import React from "react";
```

Comment:
install eslint and prettier and others, It will help with fixing by conventions. like React comes first and the rest better be orders alphabetically

Here is a blog about   those: 
https://medium.com/dooboolab/using-eslint-prettier-and-sort-imports-vscode-extensions-for-formatting-open-source-project-16edf317129d

Reply:
We already use eslint

Reply:
We already have eslint - dont think its critical that it will align with what you have on webux